backend/services/bookingService.ts

```diff
@@ -127,12 +126,6 @@ export = class BookingService {
       log.error('Notify failed: ', e)
     }
 
-    raiseAnalyticsEvent(
```

Comment:
Talked to Jonathan about this and he's not convinced it works as is, not migrated it to Tag Manager but we're happy to re-create it in Tag Manager if we do need it further down the line.